model_compression_toolkit/keras/quantizer/gradient_ptq/config_factory.py

```diff
@@ -23,22 +23,21 @@
 from model_compression_toolkit import common
 from model_compression_toolkit import keras
 from model_compression_toolkit.common.constants import THRESHOLD
-from model_compression_toolkit.keras.constants import DEPTHWISE_KERNEL, KERNEL
 from model_compression_toolkit.common.framework_info import FrameworkInfo
-
-
-MAX_LSBS_CHANGE = 8
+from model_compression_toolkit.common.gptq.gptq_config import GradientPTQConfig
 
 
 def quantization_config_builder_gptq(n: common.BaseNode,
-                                     fw_info: FrameworkInfo) -> QuantizeConfig:
+                                     fw_info: FrameworkInfo,
+                                     gptq_config: GradientPTQConfig) -> QuantizeConfig:
     """
     Build a QuantizeConfig for a node according to its quantization configuration and
     a global NoOpQuantizeConfig object.
 
     Args:
         n: Node to build its QuantizeConfig.
         fw_info: Framework information (e.g., mapping from layers to their attributes to quantize).
+        gptq_config:
```

Comment:
gptq_config: GradientPTQConfig object for building QuantizeConfigs with max LSB change mapping.